lib/install.ps1

```diff
@@ -365,7 +365,31 @@ function dl($url, $to, $cookies, $progress) {
         }
     }
 
-    $wres = $wreq.getresponse()
+    try {
+        $wres = $wreq.GetResponse()
+    } catch [System.Net.WebException] {
+        $exc = $_.Exception
+        $redirectRes = $exc.Response
+        if ($redirectRes.StatusCode -ne [System.Net.HttpStatusCode]::Redirect) {
```

Comment:
I think we should also handle:
- Moved/MovedPermanently 301
- TemporaryRedirect 307

Reply:
Good call! I've made that change.